examples/nidcpower_source_dc_voltage/_helpers.py

```diff
@@ -231,3 +232,111 @@ def use_simulation_option(default: bool) -> Callable[[F], F]:
         is_flag=True,
         help="Use simulated instruments.",
     )
+
+
+def create_driver_session(
+    measurement_service: MeasurementService,
+    pin_names: Iterable[str],
+    instrument_type_module: Any,
+    service_options: ServiceOptions,
+    instrument_type_id: str,
+) -> Tuple[Any, List[session_management.SessionInformation]]:
+    """Create and reserve driver sessions."""
+    session_management_client = nims.session_management.Client(
+        grpc_channel=measurement_service.get_channel(
+            provided_interface=nims.session_management.GRPC_SERVICE_INTERFACE_NAME,
+            service_class=nims.session_management.GRPC_SERVICE_CLASS,
+        )
+    )
+
+    with contextlib.ExitStack() as stack:
```

Comment:
This destroys the driver session before returning. 

FYI, I am planning to minimize the use of ExitStack: https://github.com/ni/measurementlink-python/issues/260

Reply:
Removed ExitStack code block